apps/backend/src/schemas/container.py

```
@@ -6,24 +6,29 @@
 from typing import Optional, Dict, Any, List
 from uuid import UUID
 from pydantic import BaseModel, Field, field_validator
-from apps.backend.src.schemas.common import PaginatedResponse, TimeRangeParams
+from apps.backend.src.schemas.common import (
+    PaginatedResponse, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)